sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -0,0 +1,460 @@
+        print("Hate severity: {}".format(response.hate_result.severity))
+    if response.self_harm_result is not None:
+        print("SelfHarm severity: {}".format(response.self_harm_result.severity))
+    if response.sexual_result is not None:
+        print("Sexual severity: {}".format(response.sexual_result.severity))
+    if response.violence_result is not None:
+        print("Violence severity: {}".format(response.violence_result.severity))
+
+
+if __name__ == "__main__":
+    analyze_image()
+```
+
+### Manage text blocklist
+
+```python
+import os
+from azure.ai.contentsafety import ContentSafetyClient
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.contentsafety.models import (
+    TextBlockItemInfo,
+    AddBlockItemsOptions,
+    RemoveBlockItemsOptions,
+    AnalyzeTextOptions,
+)
+from azure.core.exceptions import HttpResponseError
+import time
+
+
+key = os.environ["CONTENT_SAFETY_KEY"]
+endpoint = os.environ["CONTENT_SAFETY_ENDPOINT"]
+
+blocklist_name = "TestBlocklist"
+blocklist_description = "Test blocklist management."
+block_item_text_1 = "k*ll"
+block_item_text_2 = "h*te"
+input_text = "I h*te you and I want to k*ll you."
+
+# Create an Content Safety client
+client = ContentSafetyClient(endpoint, AzureKeyCredential(key))
+
+def create_or_update_text_blocklist():
+    try:
+        blocklist = client.create_or_update_text_blocklist(blocklist_name=blocklist_name, resource={"description": blocklist_description})
+        if blocklist is not None:
+            print("\nBlocklist created or updated: ")
+            print("Name: {}, Description: {}".format(blocklist.blocklist_name, blocklist.description))
+    except HttpResponseError as e:
+        print("\nCreate or update text blocklist failed: ")
+        if e.error is not None:
+            print("Error code: {}".format(e.error.code))
+            print("Error message: {}".format(e.error.message))
+            raise
+        print(e)
+        raise
+
+def add_block_items():
+    block_items = [TextBlockItemInfo(text=block_item_text_1), TextBlockItemInfo(text=block_item_text_2)]
+    try:
+        result = client.add_block_items(
+            blocklist_name=blocklist_name,
+            body=AddBlockItemsOptions(block_items=block_items),
+        )
+        if result is not None and result.value is not None:
+            print("\nBlock items added: ")
+            for block_item in result.value:
+                print("BlockItemId: {}, Text: {}, Description: {}".format(block_item.block_item_id, block_item.text, block_item.description))
+    except HttpResponseError as e:
+        print("\nAdd block items failed: ")
+        if e.error is not None:
+            print("Error code: {}".format(e.error.code))
+            print("Error message: {}".format(e.error.message))
+            raise
+        print(e)
+        raise
+
+def analyze_text_with_blocklists():
+    try:
+        print("\nWaiting for blocklist service update...")
+        time.sleep(30)
```

Comment:
Why do we need to sleep here?

Reply:
This is related to service design, when you update items in a list, the service need time to update cache and it's not taking effect immediately, so that we sleep here to avoid calling analyze without items updated.